server/app/controllers/fusor/api/customer_portal/customer_portal_proxies_controller.rb

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+module Fusor
+  class Api::CustomerPortal::CustomerPortalProxiesController < Api::V2::BaseController
+
+    before_filter :proxy_request_path, :proxy_request_body
+
+    def get
+      response = Resources::CustomerPortal::Proxy.get(@request_path)
+      logger.debug response
```

Comment:
I have a fuzzy feeling that there is something like that in Katello or Foreman to be able to log out all responses. Maybe an around filter in an app-controller or a middleware. I could not find it though, it was in the log outputs though. 
cc @tstrachota do you know by any chance?